### PR TITLE
feat: Initial version of an extension to list contract method names

### PR DESCRIPTION
**Pull request by Buckram123**
Creating extension for #70

### CODE REVIEW

File: extensions/method-names/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[package]
+name = "method-names"
```
Comment (frol) [inline extensions/method-names/Cargo.toml:2]: It should be `near-cli-method-names` to produce a binary that `near-cli-rs` can pick up when one calls `near-cli method-names`.

**Comment by frol** [comment]
@Buckram123 

1. I recommend using `interactive-clap` to build the extensions with interactivity (see #93)
2. We should not merge extensions into this repo, instead:
  2.1. setup a separate repo
  2.2. configure CI (copy `.github/workflows`)
  2.3. add the reference and a short description to near-cli-rs README

**Review by frol — CHANGES_REQUESTED** [review]

**Comment by Buckram123** [comment]
Building it [here](https://github.com/Buckram123/near-cli-method-names), working on:

- [ ] README
- [ ] Configure CI

**Comment by frol** [comment]
@Buckram123 Ouch, I somehow missed your last message. Since that time devtools team developed [NEAR ABI](https://github.com/near/abi) and [`cargo near` extension](https://github.com/near/cargo-near).

I feel `cargo near` should remain dedicated to NEAR development on Rust, so we can start thinking about `near-cli-dev` which would be a generic extension for developers and yet agnostic to the language. It will include this feature with ABI support (using it to not only list the functions but also their signatures), and [more](https://github.com/near/near-cli-rs/blob/master/docs/NEW_NEAR_CLI_INTERFACE.md#developer-extension) (the commands layout still needs to be properly designed).

**Comment by frol** [comment]
This is now implemented as `near contract inspect` command in #293

**Pull request closed without merging**